lib/openxml/docx/style.rb

```diff
@@ -84,11 +84,19 @@ def install_table_properties
       end
 
       def property_xml(xml)
+        style_property_xml(xml)
+
         return table.property_xml(xml) if table_style?
         character.property_xml(xml)
         paragraph.property_xml(xml) if paragraph_style?
       end
 
+      def style_property_xml(xml)
+        props = properties.keys.map(&method(:send)).compact
```

Comment:
Alright, well I'll merge then and we can make a mental note to look at it more later